CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ## 2.26.1 (2024-01-05)
 
 - Fix an error for `RSpec/SharedExamples` when using examples without argument. ([@ydah])
+- Fix a false positive for `RSpec/ExpectActual` when used with rspec-rails routing matchers. ([@naveg])
```

Comment:
Please move this line to the `Unreleased` section :-)

Reply:
woops, fixed